packages/@react-aria/gridlist/src/useGridListSection.ts

```diff
@@ -37,20 +42,62 @@ export interface GridListSectionAria {
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function useGridListSection<T>(props: AriaGridListSectionProps, state: ListState<T>, ref: RefObject<HTMLElement | null>): GridListSectionAria {
-  let {'aria-label': ariaLabel} = props;
+  let {'aria-label': ariaLabel, node, isVirtualized} = props;
   let headingId = useSlotId();
   let labelProps = useLabels({
     'aria-label': ariaLabel,
     'aria-labelledby': headingId
   });
+  let rowIndex;
+
+  let sumOfNodes = (node: RSNode<unknown>): number => {
```

Comment:
should this be a shared function with the logic in useGridListItem? I know they are both a little different, so it may be helpful to split these functions into multiple more specific functions

you can pass node and state.collection and anything else to the shared function(s)